<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationsWidget</class>
 <widget class="QWidget" name="OperationsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>164</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Type name</string>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 2px;</string>
       </property>
       <property name="text">
        <string>Safir.Dob.Object</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="copyNameBtn">
       <property name="toolTip">
        <string>Copy type name to clipboard.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="typeIdLabel">
       <property name="toolTip">
        <string>Type id</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 12px;
margin: 2px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>12345</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="copyTypeIdBtn">
       <property name="toolTip">
        <string>Copy type id to clipboard.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setChangesBtn">
     <property name="text">
      <string>Set changes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setAllBtn">
     <property name="text">
      <string>Set all</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteBtn">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="createReqBtn">
     <property name="text">
      <string>Create request</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="updateReqBtn">
     <property name="text">
      <string>Update request</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteReqBtn">
     <property name="text">
      <string>Delete request</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sendMsgBtn">
     <property name="text">
      <string>Send message</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sendReqBtn">
     <property name="text">
      <string>Send request</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="idLabel">
     <property name="styleSheet">
      <string notr="true">margin-top: 20px;</string>
     </property>
     <property name="text">
      <string>Handler Id</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="idEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="instanceLabel">
     <property name="styleSheet">
      <string notr="true">margin-top: 8px;</string>
     </property>
     <property name="text">
      <string>Instance Id</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="instanceEdit"/>
   </item>
   <item>
    <widget class="QToolButton" name="copyEntityIdBtn">
     <property name="text">
      <string>Copy entity id</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="liveUpdateCheckBox">
     <property name="styleSheet">
      <string notr="true">margin-top: 10px;</string>
     </property>
     <property name="text">
      <string>Live updates</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="toXmlBtn">
     <property name="text">
      <string>To XML</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="toJsonBtn">
     <property name="text">
      <string>To Json</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
